packages/xstate-react/package.json

```diff
@@ -54,7 +54,7 @@
     "url": "https://github.com/statelyai/xstate/issues"
   },
   "peerDependencies": {
-    "react": "^16.8.0 || ^17.0.0 || ^18.0.0 || ^19.0.0-0",
```

Comment:
this wasn't really necessary - the users who might still be at some React 19 beta versions will now get peer dep mismatch warnings